group_vars/location_rixbox/networks.yml

```diff
@@ -60,13 +60,12 @@ networks:
     prefix: 10.31.154.112/28
     assignments:
       rixbox-core: 1
-      rixbox-ap1: 2
-      rixbox-ap2: 3
-      rixbox-ap3: 4
-      # AirOS LiteBeam RHNK
-      rixbox-rhnk: 5
+      rixbox-nf-nw: 2
+      rixbox-nf-no: 3
+      rixbox-nf-sw: 4
+      rixbox-rhnk: 5  # AirOS 8.7.11
```

Comment:
do you really want to put airos version information here? You would need to update it, if you udate the airos antenna? ;)

Reply:
This is how I remember :grimacing:

Reply:
okay, it's up to you xP